oxidation/client/src/router/index.ts

```diff
@@ -32,7 +32,7 @@ const routes: Array<RouteRecordRaw> = [
             component: () => import('@/views/WorkspacesPage.vue')
           },
           {
-            path: '/:deviceId([a-z0-9]+@[a-z0-9]+)/workspaces/:workspaceId(\\d+)',
+            path: '/:deviceId([a-z0-9]+@[a-z0-9]+)/workspaces/:workspaceId([a-z0-9]+)',
```

Comment:
How the `EntryID` are formatted here ?

Reply:
It doesn't matter, we don't have EntryIDs yet. It's just meant to work with our mock workspaces.

Reply:
Add a `TODO` comment :thinking: ?

Reply:
it is logged in an issue, closing.